docs/docs-beta/docusaurus.config.ts

```diff
@@ -12,7 +12,8 @@ const config: Config = {
   baseUrl: '/',
   onBrokenLinks: 'throw',
   onBrokenMarkdownLinks: 'throw',
-  onBrokenAnchors: 'throw',
+  // TODO - reintroduce when API doc anchors are fixed
+  onBrokenAnchors: 'warn',
```

Comment:
temporarily "warn" on broken anchor links, there are many in api docs.

Reply:
reminder that the longer this TODO takes, the more painful you will get when you also need to fix many real broken anchors in the future. but as we're going through several simultaneous efforts to rework nav/pages, it might be a good idea to do a pass in the end when things are settled a bit anyways 🤷‍♀️